### ISSUE TITLE
Need dynamic wmenu_type

### ISSUE THREAD

**Issue by mikeotoole** [bmoredrew/WeedMaps-Menu-WordPress#3]
``var wmenu_type = 'dispensaries'`` now needs to be set to either deliveries or dispensaries depending on the users listing type. Without this deliveries will see a 404 page.

**Comment by mikeotoole** [comment]
I believe this is the issue hitbridge5 is having here: https://wordpress.org/support/topic/weedmaps-menu-not-showing-up-on-page

**Comment by bmoredrew** [comment]
Yeah, I just recently figured that out, going to add this as a shortcode attribute. I don't have access to any active WM's accounts, so when they roll out changes I'm trying to reverse engineer it. 

Thanks!!
